### PR TITLE
[zh-cn] Add page for docs/tasks/administer-cluster/verify-signed-artifacts.md

### PR DESCRIPTION
**Pull request by Zhuzhenghao**
Hello everyone,

Chinese translation for the [Verify Signed Kubernetes Artifacts](https://kubernetes.io/docs/tasks/administer-cluster/verify-signed-artifacts/) page.

Related issues: https://github.com/kubernetes/website/issues/38480

### CODE REVIEW

**Comment by mengjiao-liu** [comment]
Please see https://github.com/kubernetes/website/issues/38577#issuecomment-1360821830

File: content/zh-cn/docs/tasks/administer-cluster/verify-signed-artifacts.md [hunk 1]
```diff
@@ -33,6 +33,49 @@ You will need to have the following tools installed:
...
+<!-- 
+## Verifying binary signatures
+
+The Kubernetes release process signs all binary artifacts (tarballs, SPDX files,
+standalone binaries) by using cosign's keyless signing. To verify a particular
+binary, retrieve it together with its signature and certificate: -->
+
+## 验证二进制签名 {#verifying-binary-signatures}
+
+Kubernetes 发布过程使用 cosign 的无密钥签名对所有二进制制品（tarballs、SPDX 文件、独立二进制文件）进行了签名。要验证特定的二进制文件，在获取这个二进制文件的时候，可以配套获取签名和证书的信息，参考下面的示例：
```
Comment (howieyuen) [inline content/zh-cn/docs/tasks/administer-cluster/verify-signed-artifacts.md:45]:
```suggestion
Kubernetes 发布过程对所有二进制工件（压缩包、SPDX 文件、 独立的二进制文件）通过使用 cosign 的无密钥签名。
要验证一个特定的二进制文件，连同其签名和证书一起获取：
```

File: content/zh-cn/docs/tasks/administer-cluster/verify-signed-artifacts.md [hunk 2]
```diff
@@ -33,6 +33,49 @@ You will need to have the following tools installed:
...
+```shell
+cosign verify-blob "$BINARY" --signature "$BINARY".sig --certificate "$BINARY".cert
+```
+
+{{< note >}}
+<!-- 
+To learn more about keyless signing, please refer to [Keyless
+Signatures](https://github.com/sigstore/cosign/blob/main/KEYLESS.md#keyless-signatures).
+-->
+
```
Comment (howieyuen) [inline content/zh-cn/docs/tasks/administer-cluster/verify-signed-artifacts.md:74]:
```suggestion
```

File: content/zh-cn/docs/tasks/administer-cluster/verify-signed-artifacts.md [hunk 3]
```diff
@@ -33,6 +33,49 @@ You will need to have the following tools installed:
...
+想要进一步了解 `KEYLESS`，请参考
```
Comment (howieyuen) [inline content/zh-cn/docs/tasks/administer-cluster/verify-signed-artifacts.md:75]:
```suggestion
想要进一步了解无密钥签名，请参考
```

**Review by howieyuen — CHANGES_REQUESTED** [review]

File: content/zh-cn/docs/tasks/administer-cluster/verify-signed-artifacts.md (hunk 1 above)
Comment (Zhuzhenghao) [reply content/zh-cn/docs/tasks/administer-cluster/verify-signed-artifacts.md]: Thank you for taking the time to review my pull request and for providing these suggestions. I have made the requested changes to the code . Please let me know if you have any further questions or concerns. Thank you again for your help!

Comment (Zhuzhenghao) [reply content/zh-cn/docs/tasks/administer-cluster/verify-signed-artifacts.md]: Thank you for taking the time to review my pull request and for providing these suggestions. I have made the requested changes to the code . Please let me know if you have any further questions or concerns. Thank you again for your help!

File: content/zh-cn/docs/tasks/administer-cluster/verify-signed-artifacts.md (hunk 3 above)
Comment (Zhuzhenghao) [reply content/zh-cn/docs/tasks/administer-cluster/verify-signed-artifacts.md]: Thank you for taking the time to review my pull request and for providing these suggestions. I have made the requested changes to the code . Please let me know if you have any further questions or concerns. Thank you again for your help!

File: content/zh-cn/docs/tasks/administer-cluster/verify-signed-artifacts.md [hunk 4]
```diff
@@ -33,6 +33,48 @@ You will need to have the following tools installed:
...
+    "$BINARY.sig"
+    "$BINARY.cert"
+)
+
+for FILE in "${FILES[@]}"; do
+    curl -sSfL --retry 3 --retry-delay 3 "$URL/$FILE" -o "$FILE"
+done
+```
+
+<!-- Then verify the blob by using `cosign`: -->
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/administer-cluster/verify-signed-artifacts.md:62]: 请将 `<!--`、`-->` 放到独立的行，这样便于将来逐行比较变更。

```suggestion
<!--
Then verify the blob by using `cosign`:
-->
```

File: content/zh-cn/docs/tasks/administer-cluster/verify-signed-artifacts.md [hunk 5]
```diff
@@ -33,6 +33,48 @@ You will need to have the following tools installed:
...
+然后使用 `cosign` 验证 blob：
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/administer-cluster/verify-signed-artifacts.md:63]:
```suggestion
然后使用 `cosign` 验证二进制文件：
```

File: content/zh-cn/docs/tasks/administer-cluster/verify-signed-artifacts.md [hunk 6]
```diff
@@ -33,6 +33,48 @@ You will need to have the following tools installed:
 - `cosign`（[安装指南](https://docs.sigstore.dev/cosign/installation/)）
 - `curl`（通常由你的操作系统提供）
 
+<!-- 
+## Verifying binary signatures
+
+The Kubernetes release process signs all binary artifacts (tarballs, SPDX files,
+standalone binaries) by using cosign's keyless signing. To verify a particular
+binary, retrieve it together with its signature and certificate: -->
```
[the remark this answers is not in the document]
Comment (Zhuzhenghao) [reply content/zh-cn/docs/tasks/administer-cluster/verify-signed-artifacts.md:41]: Thank you for taking the time to review my pull request and for providing these suggestions. I have made the requested changes to the code . Please let me know if you have any further questions or concerns. Thank you again for your help!

File: content/zh-cn/docs/tasks/administer-cluster/verify-signed-artifacts.md [hunk 7]
```diff
@@ -33,6 +33,48 @@ You will need to have the following tools installed:
...
+
+## 验证二进制签名 {#verifying-binary-signatures}
+
+Kubernetes 发布过程对所有二进制工件（压缩包、SPDX 文件、 独立的二进制文件）通过使用 cosign 的无密钥签名。要验证一个特定的二进制文件，连同其签名和证书一起获取：
```
[the remark this answers is not in the document]
Comment (Zhuzhenghao) [reply content/zh-cn/docs/tasks/administer-cluster/verify-signed-artifacts.md]: Thank you for taking the time to review my pull request and for providing these suggestions. I have made the requested changes to the code . Please let me know if you have any further questions or concerns. Thank you again for your help!

File: content/zh-cn/docs/tasks/administer-cluster/verify-signed-artifacts.md (hunk 4 above)
Comment (Zhuzhenghao) [reply content/zh-cn/docs/tasks/administer-cluster/verify-signed-artifacts.md]: Thank you for taking the time to review my pull request and for providing these suggestions. I have made the requested changes to the code . Please let me know if you have any further questions or concerns. Thank you again for your help!

**Review by howieyuen — COMMENTED** [review]
/lgtm

**Pull request merged**